fullstack-cert/js-projects/pantry-lab/user-stories.md

```diff
@@ -0,0 +1,5 @@
+1. You should implement parseShipment(rawData) that returns an array of { sku, name, qty, expires } objects.
```

Comment:
We will need a few sentences explaining the basic premise of what this project is about. Because with just user stories, campers won't know what this project is about. Providing them a general idea of what to build will help them a lot. You can review other JS labs for an idea on introductions.